control-plane/tests/test_service_persistence_ingest_case_lifecycle.py

```
@@ -3559,6 +3559,137 @@ def test_service_analyst_queue_sorts_unknown_last_seen_after_real_timestamps(
             ("alert-known-last-seen", "alert-unknown-last-seen"),
         )
 
+    def test_service_marks_queue_lanes_for_mismatch_stale_and_degraded_context(
+        self,
+    ) -> None:
+        store, _ = make_store()
```

Ruff (0.15.11)

[error] Undefined name `make_store`

(F821)